vscode_extension/src/AacTaskProvider.ts

```diff
@@ -36,6 +39,13 @@ function execShell(command: string, options: cp.ExecOptions): Promise<{ stdout:
     });
 }
 
+function maybeConvertBufferToString(arg: string | Buffer, encoding: BufferEncoding = "utf-8"): string {
```

Comment:
convertBufferToString should be adequate. 
```suggestion
function convertBufferToString(arg: string | Buffer, encoding: BufferEncoding = "utf-8"): string {
```

Reply:
Resolved in 829c8346988874e810ada2d743cb0127dbe593c4